lib/mayaUsd/fileio/writeJobContext.cpp

```diff
@@ -406,15 +404,18 @@ bool UsdMayaWriteJobContext::_NeedToTraverse(const MDagPath& curDag) const
 
         const std::string mayaTypeName(depFn.typeName().asChar());
 
-        if (mArgs.excludeExportTypes.count(TfToken("Meshes")) != 0) {
+        if ((mArgs.excludeExportTypes.count(TfToken("Meshes")) != 0)
+            || (mArgs.excludeExportTypes.count(TfToken("meshes")) != 0)) {
```

Comment:
I think it would be cleaner and more general to lowercase both strings and compare those.